examples/with-uniswap/src/scripts/univ3-swap.ts

```diff
@@ -0,0 +1,55 @@
+import { ethers } from "ethers";
```

Comment:
Nit: let's name this file `index.ts` or `main.ts` (such that it's easy to discover what the entry point is)